### PR TITLE
Boarding

### LINKED ISSUE #1434: Galactica, XO office issues
XO office doesn't have spare pagers. One pilot refused to leave the line a la papers please, I sent them on their way with a global pager 
XO office runs out of power in about 5 minutes and doesn't recharge after engineering does their thing

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

https://www.youtube.com/watch?v=WOeYPpOblAw

## Why It's Good For The Game

Sharpe was a really good TV show, not even just for its time but even now. Oh, also Karmic says the books are good too

fixes #1434

## Changelog
:cl: Kmc2000
add: Redid the entire squads system from the ground up. They're more useful now
add: You can now board and commandeer vessels! Check out the tactical console
add: Added the marine role! Comes with a dropship and a crayon addiction
del: Assistant role replaced by marine
/:cl:

### CODE REVIEW

**Comment by AtlasRules** [comment]
YAAAAAAAAAAAAAAAAAAAAARRRRRRRRRRRRRRRRRRRRR

**Comment by bluey89962** [comment]
Security Officer hours were wiped, my guess is that this is due to the name change. Dunno if any of the other changed jobs got affected in the same way

**Comment by ilikesaying** [comment]
SPHESS MUHREENS

**Comment by benbot16** [comment]
Would it be possible to use the old squad icons rather than the ones we have now? IMO they look a lot better. Might not work as well though.

File: nsv13/code/modules/overmap/ftl.dm [hunk 1]
```diff
@@ -133,7 +133,7 @@
 		if(!istype(AM, /obj/structure/overmap))
 			continue
 		var/obj/structure/overmap/OM = AM
-		if(!OM.operators.len || OM.ai_controlled)	//AI ships / ships without a pilot just get put in stasis.
+		if(!OM.occupying_levels?.len)	//AI ships / ships without a pilot just get put in stasis.
```
Comment (DeltaFire15) [inline nsv13/code/modules/overmap/ftl.dm:136]: :)
```suggestion
		if(!OM.occupying_levels?.len || OM.ai_controlled)	//AI ships / ships without a pilot just get put in stasis.
```

**Comment by SerynEngi** [comment]
As this appears to change permissions, please fix default pilot permissions to have everything on high pop. As it is, they dont have sufficient permissions to complete their jobs.

File: nsv13/code/modules/jobs/job_types/pilot.dm [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+	access = list(ACCESS_MAINT_TUNNELS, ACCESS_MUNITIONS, ACCESS_MUNITIONS_STORAGE, ACCESS_COMBAT_PILOT, ACCESS_TRANSPORT_PILOT, ACCESS_HANGAR)
+	minimal_access = list(ACCESS_MAINT_TUNNELS, ACCESS_MUNITIONS, ACCESS_HANGAR)
+	paycheck = PAYCHECK_MEDIUM
+	paycheck_department = ACCOUNT_MUN
+	mind_traits = list(TRAIT_MUNITIONS_METABOLISM)
+
+	display_order = JOB_DISPLAY_ORDER_PILOT
+
+/datum/job/officer/after_spawn(mob/living/carbon/human/H, mob/M)
```
Comment (DeltaFire15) [inline nsv13/code/modules/jobs/job_types/pilot.dm:26]:
```suggestion
/datum/job/pilot/after_spawn(mob/living/carbon/human/H, mob/M)
```

**Comment by Cenrus** [comment]
the marine guns are unacceptably overpowered and antags are getting shafted by the marines' easily avaliable nukie tier weaponry and armour.

**Comment by covertcorvid** [comment]
> 
> 
> Security Officer hours were wiped, my guess is that this is due to the name change. Dunno if any of the other changed jobs got affected in the same way

Once the changes are final, we have a way to copy over the hours for both security officers and fighter pilots.

**Comment by Kmc2000** [comment]
@DeltaFire15 ty bb

**Comment by SerynEngi** [comment]
As we're noting things missed with pilots. Request fighter light cannon ammo be on the Tycoon by default (instead of having to round-start order it), and maybe a few of the light fighters swapped out with heavy fighters (and 530mm torps) given that we have 
fighter heavy cannon ammo round-start.

**Comment by covertcorvid** [comment]
Whoops, I thought that would push to my fork, not straight to the PR. Will fix my remote but uh, this should some some CI problems so we can see where the real problems are

**Comment by Kmc2000** [comment]
Sorry in advance whoever rebases us next 🔫

**Pull request merged**